bezier-rs/docs/interactive-docs/wasm/src/subpath.rs

```diff
@@ -0,0 +1,48 @@
+use bezier_rs::subpath::{ManipulatorGroup, SubPath, ToSVGOptions};
+use glam::DVec2;
+use wasm_bindgen::prelude::*;
+
+/// Wrapper of the `SubPath` struct to be used in JS.
+#[wasm_bindgen]
+pub struct WasmSubPath(SubPath);
+
+const SVG_OPEN_TAG: &str = r#"<svg xmlns="http://www.w3.org/2000/svg" width="200px" height="200px">"#;
+const SVG_CLOSE_TAG: &str = "</svg>";
+
+#[wasm_bindgen]
+impl WasmSubPath {
+	/// Expect js_points to be a list of 3 pairs.
+	pub fn from_triples(js_points: &JsValue, closed: bool) -> WasmSubPath {
```

Comment:
@TrueDoctor can this be enforced in the type with something like `js_points: &[JsValue; 3]`?

Reply:
I've fixed the comment here as `js_points` is not actually just a triple, its actually a list of triples (which is then converted to a list of `ManipulatorGroups`). I experimented here a bit to see if we could get the type enforced, but WASM complains about all vectors unless they're vectors of primitive types, so I don't believe there's an good way to enforce the type properly here.